react/GiftText.tsx

```diff
@@ -35,15 +34,37 @@ const ProductGiftText: StorefrontFunctionComponent<Props> = ({
 
   return (
     <span className={handles.productGiftText}>
-      <IOMessage id={translatableText} values={values} />
+      <IOMessage id={text} values={values} />
     </span>
   )
 }
 
-ProductGiftText.schema = {
-  title: 'ProductGifts',
-  description: 'A test, for now.',
-  type: 'Object',
+/**
+ * These messages are used in contentSchema.json and need to be defined for the messages
```

Comment:
Isn't better define a `context.json` instead of polluting the code file with these messages?